src/Metadata/Resource/Factory/YamlResourceNameCollectionFactory.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ApiPlatform\Core\Metadata\Resource\Factory;
+
+use ApiPlatform\Core\Annotation\Resource;
+use ApiPlatform\Core\Metadata\Resource\ResourceNameCollection;
+use ApiPlatform\Core\Metadata\Resource\ConfigurationMap;
+use Symfony\Component\Yaml\Parser as YamlParser;
+
+/**
+ * Creates a resource name collection from {@see Resource} configuration files.
+ *
+ * @author Antoine Bluchet <soyuka@gmail.com>
+ */
+final class YamlResourceNameCollectionFactory implements ResourceNameCollectionFactoryInterface
+{
+    /** @var Symfony\Component\Yaml\Parser **/
```

Comment:
Do not use the single-line docblock for properties. (It should not use 2 asterisks for closing anyway... :laughing:)